src/Components/Profile/EditProfile.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import toastr from "toastr";
+
+export class EditProfile extends Component {
+	constructor(props) {
```

Comment:
Using a constructor when initialising state is not required, could you reformat it to look like: 
state = {
username: ..........,
}

Reply:
@shakirandagire I worked on this